.travis.yml

```diff
@@ -27,8 +26,9 @@ install:
   - pip install sphinx sphinx-bootstrap-theme coveralls ipython[all]==2.4.1
   - travis_retry pip install .
 script:
-  - export QIITA_CONFIG_FP=`pwd`/qiita_core/support_files/config_test_travis.cfg
-  - export MOI_CONFIG_FP=`pwd`/qiita_core/support_files/config_test_travis.cfg
+  - export MOI_CONFIG_FP=`pwd`/qiita_core/support_files/config_test.cfg
+  - if [ ${TRAVIS_PULL_REQUEST} == "false" ]; then export QIITA_CONFIG_FP=`pwd`/qiita_core/support_files/config_test_travis.cfg; fi
```

Comment:
I could do this but it will make a really long single line, which I'm not a fan of.

See: https://groups.google.com/forum/#!topic/travis-ci/uaAP9zEdiCg